src/styles/card.scss

```diff
@@ -0,0 +1,112 @@
+@import '../styles/variables';
+
+.card {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  max-width: 166px;
+  border-radius: $border-radius;
+  padding: 32px 16px 16px;
+  border: 1px solid #f3f3f3;
+
+  &__image {
+    width: 160px;
+  }
```

Comment:
Magic number '160px': if this width is related to the `max-width` of `.card`, it should be derived from variables or calculated to ensure responsiveness and maintainability.